### ISSUE TITLE
[Patch] Correctly set keyboard focus

### ISSUE THREAD

**Issue by SDLBugzilla** [libsdl-org/SDL#2461]
# This bug report was migrated from our old Bugzilla tracker.

These attachments are available in the static archive:

* [Add missing parameter to message (haiku_keyboardfocus.patch, text/plain, 2017-05-28 14:22:09 +0000, 412 bytes)](https://bugzilla.libsdl.org/attachment.cgi?id=2747)

**Reported in version:** HG 2.0
**Reported for operating system, platform:** Haiku, x86

# Comments on the original bug report:

On 2017-05-28 14:22:09 +0000, Kai Sterker wrote:

> Created attachment 2747
> Add missing parameter to message
> 
> The message sent upon the window being activated or deactivated, to trigger the call to SDL_SetKeyboardFocus was missing a mandatory parameter. So keyboard focus was never properly set.

On 2017-05-28 19:36:33 +0000, Ryan C. Gordon wrote:

> 
> This patch is now https://hg.libsdl.org/SDL/rev/57e3891f1ef8, thanks!
> 
> --ryan.
